tests/fast/ProtocolVersion.toml

```diff
@@ -0,0 +1,7 @@
+protocolVersion = 0x0FDB00B070010000
```

Comment:
I was imagining a boolean, something like `startIncompatibleProcess`, which instructs the simulator to start an extra process with a version higher than currentProtocolVersion